theseus/extlib/cusolver_lu_solver.cpp

```diff
@@ -26,6 +26,8 @@ enum Ordering {
 
 struct CusolverLUSolver {
 
+	~CusolverLUSolver();
```

Comment:
Maybe add a small unit test if necessary?

Reply:
Sorry for the late answer... yeah I don't know of an easy way to test this. I would make sure the code does the right thing (destroying all contexts that might have been created) and then rely of RAII logic of C++